ui/src/Manager.cs

```diff
@@ -233,5 +245,57 @@ public static void InitializeSettings()
         {
             Settings = new Settings(ProductConstants.SettingsFile);
         }
+
+        /// <summary>
+        /// Initializes cache for avatar image.
+        /// </summary>
+        public static void InitializeCache()
+        {
+            Cache = MemoryCache.Default;
+
+            if (Account.LoginState == FxA.LoginState.LoggedIn)
+            {
+                CacheItemPolicy policy = new CacheItemPolicy();
+
+                Task.Run(() =>
+                {
+                    Application.Current.Dispatcher.Invoke(() =>
+                    {
+                        var image = new BitmapImage();
+
+                        if (Account.Config.FxALogin.User.Avatar != null)
+                        {
+                            HttpWebRequest req = (HttpWebRequest)WebRequest.Create(Account.Config.FxALogin.User.Avatar);
+
+                            try
+                            {
+                                using (HttpWebResponse response = (HttpWebResponse)req.GetResponse())
+                                {
+                                    image = new BitmapImage(new Uri(Account.Config.FxALogin.User.Avatar));
+                                }
+                            }
+                            catch (WebException)
+                            {
+                                image = new BitmapImage(new Uri(Path.Combine(Environment.CurrentDirectory, @"..\..\..\src\UI\Resources\Icons\Generic\default.png")));
```

Comment:
Could you rename `default.png` to something more descriptive, like `default-avatar.png`?